data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/put_to_recall_and_modify.cfg

```diff
@@ -5,7 +5,7 @@
 # which needs to be handled within the WML API.
 
 #####
-# API(s) being tested: [put_to_recall_list], [modify_unit], [role][auto_recall]
+# API(s) being tested: [role][auto_recall]
```

Comment:
`[modify_unit]` should be in this list - the bug that was fixed alongside adding this test is that the `[modify_unit]` put Charlie back on the map instead of the recall list.

Reply:
I'd say this should just be `[modify_unit]` then.

Reply:
Changed to `[modify_unit]`.

Reply:
I've marked the other remaining "API(s) being tested" lines as resolved, but this one should be reverted. I looked at whether it's possible to split it into two separate tests to fit the "it's not ideal to have a single test that's testing APIs from multiple categories" comment, however the only difference between the two tests would be that one ends before the `[role]` assignment.

The bug that prompted this test to be written requires a unit to be on the recall list with `x,y` values which are valid to be on-map; which requires using `[put_to_recall_list]` instead of creating it directly with `[unit]x,y=recall,recall`. Because of this, if it were split into two tests without using Lua, then both of them have to start at the "Create a unit on the map, belonging to side 1" step.

Reply:
Updated.